src/toxcore/dht/dht_node.rs

```diff
@@ -94,8 +94,8 @@ impl DhtNode {
     }
 
     /// return SocketAddr for DhtNode
-    pub fn get_socket_addr(&self, is_ipv6_mode: bool) -> Option<SocketAddr> {
-        if is_ipv6_mode {
+    pub fn get_socket_addr(&self, is_ipv6_enabled: bool) -> Option<SocketAddr> {
+        if is_ipv6_enabled {
```

Comment:
See logic https://github.com/tox-rs/tox/pull/183/files#diff-77f898031e13d02915c0d710043ed97bR142 

Please don't duplicate code. It is better to do PRs 1 by 1